odo/backends/sql.py

```diff
@@ -103,7 +103,7 @@
     mssql.UNIQUEIDENTIFIER: string,
     # The SQL Server TIMESTAMP value doesn't correspond to the ISO Standard
     # It is instead just a binary(8) value with no relation to dates or times
-    mssql.TIMESTAMP: bytes_,
+    # mssql.TIMESTAMP: bytes_,
```

Comment:
Maybe we can replace the type sqlalchemy returns on a MSSQL connection? http://docs.sqlalchemy.org/en/latest/core/custom_types.html#replacing-processors

Just make a custom class like:

```
class MSSQLTimestamp(mssql.TIMESTAMP):
    pass
```